tests/unit_tests/client/proxy/http.cc

```diff
@@ -57,4 +57,16 @@ BOOST_AUTO_TEST_CASE(ok) {
   BOOST_CHECK(tmp.HandleData(tmpData));
 }
 
+BOOST_AUTO_TEST_CASE(jumpI2PAddressHelper) {
+  kovri::client::HTTPMessage tmp;
+  // Jump service GET parameter
+  std::string tmpData = "GET stats.i2p?i2paddresshelper=";
+  // Base64-encoded I2P address
+  tmpData+="WED~HHcaMK5GdfwrIOnDLj27Waso9~UUM4PBy~Dqn8-chmJwb1yg3l7WTsMRKkAkMs7vAk3pWOLvxZPRxGP3jP1ELNUg3mL2Dvj8euij8efF38fxzUcgNQ-DpFR40D44nn2si9COqIDjm1njixVBOy7ab4QWFSdE5DgEGOQxjkRiAU5K5vNnq6iFT~idukr1PASA3P5KWDu4s9g~S7k9FHev~HPhUArbuS6U75YlKjQXxhZhxDzdotZ~xUS2ZJcZlsH~73OO6zL3FgCs4JlDkklgbwarK57F9Auys6mRC1KkYgmavE~w6IA8kDp4IbGnhb2Rtcfb0SZoVtifVNkflvXX8bjkaNUAZtjmRKFt9I47pHSrdNfpgCV8a5kFl1QC5Li8OD3YmZg3kgX8Q0uc4IBtTzFvyT73hlIa~guoYYsstVJ58el01RPVYqJeEm1hlxloGSgm6WheFfY664IN7J92dhZvmL4DdZdIgJwXwjKM5QCcwpp2b~yYyY~XUjSCBQAEAAcAAA==";
+  tmpData+=" HTTP/1.1\r\n\r\n";
```

Comment:
Please, read the style guide. Our upstream style guide is clearly documented. The following changes apply to the entire test-case and in any new work:

1. Don't use camelCase
2. Use appropriate spacing
3. Use appropriate names
4. Run `clang-format`

Reply:
Apologies for the misstep, failed to reread the style guide and went off surrounding style.

I'll make the changes, and keep these formatting changes in their own commit.